packages/dx-react-grid/docs/guides/data-types.md

```diff
@@ -20,7 +20,7 @@ const rows = [
 ];
 const columns = [
   { name: 'product', title: 'Product' },
-  { name: 'amount', title: 'Sale Amount', dataType: 'currency' },
+  { name: 'price', title: 'Sale Price', dataType: 'currency' },
```

Comment:
Let's keep this piece of code consistent with the [demo](https://github.com/DevExpress/devextreme-reactive/blob/v1.0.0-beta.1/packages/dx-react-demos/src/material-ui/data-types/formatters.jsx#L42).

I think it would be better if we rename `price` to `amount` in the following code sample:

```js
const rows = [
  { product: 'SolarOne', price: '3039' },
];
```